api/autoname_test.go

```diff
@@ -337,3 +337,22 @@ func (suite *AutonameSuite) TestNameByTag() {
 	suite.Require().Nil(err)
 	suite.Len(names, 8, "len(names)")
 }
+
+/*
```

Comment:
if need anything test here?

Reply:
Sure you need to test. 
What if someone change something by mistake ? you go and manually fix the DB ?!
Tests ensure the code stays as intended.

Reply:
added